### PR TITLE
shortit: project and pylint setup

### PR DESCRIPTION
**Pull request by DOMINION-JOHN1**
Created a directory for the project , installed the necessary dependencies in the requirement.txt file. Also ran pylint on the main.py file and it ran successfully.

### CODE REVIEW

File: .idea/.gitignore
Comment (greatertomi) [inline .idea/.gitignore:1]: You should not push .idea to github. Create .gitignore and add it there.

File: requirements.txt
Comment (greatertomi) [inline requirements.txt:1]: I'm suprised that the packages listed here are over 200. How did you generate this file? `pip freeze > requirements.txt`

File: .idea/.gitignore
Comment (DOMINION-JOHN1) [reply .idea/.gitignore:1]: Done!

File: venv/Scripts/python.exe
[the remark this answers is not in the document]
Comment (DOMINION-JOHN1) [reply venv/Scripts/python.exe:1]: Done!

File: requirements.txt
Comment (DOMINION-JOHN1) [reply requirements.txt:1]: I used pip freeze. I have corrected now and just added the dependencies needed for now

**Review by greatertomi — APPROVED** [review]

**Comment by greatertomi** [comment]
Nice setup. Please use "squash and merge"

**Comment by DOMINION-JOHN1** [comment]
Apologies , just getting this now. I  merged instead of using ""squash and
merge"

**Pull request merged**